client/App/Containers/Data/DataScreen.js

```diff
@@ -50,10 +73,18 @@ class WifiListScreen extends React.Component {
               {
                 this.props.wifiList.map((net, i) => (
                   <Text key={`net_${i}`} style={Style.result}>
-                    {i} - ssid: {net.SSID}, level: {net.level}
+                    {i+1}. ssid: {net.SSID}, RSSI: {net.level}
```

Comment:
While you're at it, let's use BSSID instead of SSID

Reply:
SSID is more readable for the user, so I think it's better to use it for the scree.